html/inc/user.inc

```diff
@@ -283,6 +283,19 @@ function show_user_info_private($user) {
             tra("Account keys"),
             "<a href=\"weak_auth.php\">".tra("View")."</a>"
         );
+
+        $config = get_config();
+        $keydir = parse_config($config, "<key_dir>");
+        $private_key_path = "file://$keydir/account_ownership_private.pem";
```

Comment:
These three lines are duplicated with other code and should be consolidated to a single location (maybe create inc/account_ownership.inc?)